controllers/app.controllers.js

```diff
@@ -88,3 +89,14 @@ exports.patchVotes = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteComments = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeComments(comment_id)
+    .then((response) => {
+      res.status(204).send({ response });
```

Comment:
You shouldn't need to send anything back from a delete request, in fact nothing will come back 
So you can effectively remove this from what is sent (i.e. `send()`)